src/index.mjs

```diff
@@ -27,78 +35,162 @@ function directoryHasPackageJson(dir) {
+			DEFAULT_CONFIG_CONTENT.split('\n')
+				.map((line) => `  ${line}`)
+				.join('\n'),
+		),
+	);
+
+	console.info(
+		stripIndent(`
+			Then add the related dependencies using:
+
+			  ${picocolors.dim(`${pkgManager} ${ADD_COMMAND}`)}
+		`),
+	);
+
+	let proceed;
+
+	try {
+		proceed = await confirm({
+			message: 'Continue?',
+		});
+	} catch (error) {
+		if (error instanceof Error && error.name === 'ExitPromptError') {
+			// silence
+		} else {
+			throw error;
+		}
+	}
+
+	if (!proceed) {
+		cancelSetup();
+	}
 }
 
 /**
  * @param {string} cwd
  * @param {string} pkgManager
  */
-function createConfig(cwd, pkgManager) {
+async function createConfig(cwd, pkgManager) {
 	const spinner = ora('Creating config...').start();
-	const existingConfig = getExistingConfigInDirectory();
+	const existingConfig = await getExistingConfigInDirectory();
 
 	if (existingConfig !== null) {
 		const basename = path.basename(existingConfig.filepath);
 		const failureMessage =
 			basename === 'package.json'
-				? 'The "stylelint" config in "package.json" already exists.'
-				: `The "${basename}" config already exists.`;
+				? 'A "stylelint" config in "package.json" already exists.'
+				: `A "${basename}" config already exists.`;
 
-		spinner.fail(`Failed to create config:\n${failureMessage} Remove it and then try again.`);
-		process.exit(1);
+		spinner.fail();
+		cancelSetup(`${failureMessage} Remove it and then try again.`);
 	}
 
 	if (!directoryHasPackageJson(cwd)) {
-		spinner.fail(
-			`Failed to create config:\npackage.json was not found. Run "${pkgManager} init" and then try again.`,
-		);
-		process.exit(1);
+		spinner.fail();
+		cancelSetup(`A "package.json" was not found. Run "${pkgManager} init" and then try again.`);
 	}
 
 	try {
-		fs.writeFileSync(DEFAULT_CONFIG_FILE, '{ "extends": ["stylelint-config-standard"] }');
+		fs.writeFileSync(DEFAULT_CONFIG_FILE, DEFAULT_CONFIG_CONTENT);
 	} catch (error) {
-		spinner.fail(`Failed to create config:\n${error}`);
-		process.exit(1);
+		spinner.fail();
+		cancelSetup(error instanceof Error ? error.message : String(error));
 	}
 
-	spinner.succeed(`Created ${DEFAULT_CONFIG_FILE}.`);
+	spinner.succeed(`Created ${DEFAULT_CONFIG_FILE}`);
 }
 
 /**
  * @param {string} cwd
  * @param {string} pkgManager
  */
-async function installPackages(cwd, pkgManager) {
-	const spinner = ora('Installing packages...').start();
+async function addPackages(cwd, pkgManager) {
+	const spinner = ora('Adding packages...').start();
 
 	try {
-		await execa(
-			pkgManager,
-			[`${getInstallCommand(pkgManager)}`, '-D', 'stylelint', 'stylelint-config-standard'],
-			{ cwd },
-		);
+		await execa(pkgManager, [...ADD_COMMAND.split(' ')], {
+			cwd,
+		});
 	} catch (error) {
-		spinner.fail(`Failed to install packages:\n${error}`);
-		process.exit(1);
+		spinner.fail();
+		cancelSetup(error instanceof Error ? error.message : String(error));
 	}
 
-	spinner.succeed('Installed packages.');
+	spinner.succeed('Added packages');
```

Comment:
[suggestion] How about emphasizing the package names, as well?

If doing so, we'd like to make a list of the packages like so:

```ruby
const ADD_PACKAGES = ['stylelint', 'stylelint-config-standard'];
const ADD_COMMAND = `add -D ${ADD_PACKAGES.join(' ')}`;
```

```suggestion
	spinner.succeed(
		`Added packages ${ADD_PACKAGES.map((p) => picocolors.greenBright(p)).join(', ')}`,
	);
```